README.md

```diff
@@ -11,19 +11,9 @@ Policy-based control for Kubernetes deployments.
 
 ## Deployment Guide
 
-### Hello World
+Both OPA and `kube-mgmt` can be installed using Helm chart.
 
-1. Create a new Namespace to deploy OPA into:
-
-    ```bash
-    kubectl create namespace opa
-    ```
-
-1. Create a new Deployment that includes OPA and `kube-mgmt` (`manifests/deployment.yml`):
-
-    ```bash
-    kubectl -n opa create -f https://raw.githubusercontent.com/open-policy-agent/kube-mgmt/master/manifests/deployment.yml
-    ```
+1. Follow [instructions](charts/opa/README.md) to install it into K8s cluster.
```

Comment:
We should keep both ways of using `kube-mgmt` with and without Helm. In this way there are options users are aware of. WDYT ?

Reply:
1. Pure manifests are non customizable.
2. Current manifests are no usable because of incorrect RBAC
3. Pure manifests will always require customization, that is ambigous
4. So, I believe helm should be used only. 
5. For pure manifest - it is easy to helm template and get manifests

Reply:
It's fine to use Helm but there are places where we use manifests eg. in the [OPA-Kube tutorial](https://www.openpolicyagent.org/docs/latest/kubernetes-tutorial/). For those that don't use helm and want a quick way just to play around with `kube-mgmt`, the existing manifest is useful. If that's not working we should fix it or at least file an issue to fix it. I don't think we should be removing the exiting examples.